.github/workflows/e2e-docusaurus-workflow.yml

```diff
@@ -24,12 +24,20 @@ jobs:
     - name: 'Running the integration test'
       run: |
         source scripts/e2e-setup-ci.sh
+
+        # TODO: Remove once https://github.com/yarnpkg/berry/blob/bd146ccddf95aae9c99c0c48d86b1d8997f1dccf/scripts/e2e-setup-ci.sh#L31-L38 is fixed
+        YARN_PNP_ENABLE_ESM_LOADER=true
+
```

Comment:
I think the comment here has been fixed in 17.4.0 ([PR](https://github.com/nodejs/node/issues/37782)), correct? Perhaps we should just upgrade the Node version [here](https://github.com/yarnpkg/berry/blob/master/.github/actions/prepare/action.yml#L6)?

https://github.com/yarnpkg/berry/blob/bd146ccddf95aae9c99c0c48d86b1d8997f1dccf/scripts/e2e-setup-ci.sh#L31-L38

Reply:
Assuming you meant to link to https://github.com/nodejs/node/pull/41221 then yes, it has been fixed in 17.4.0 but I don't want to upgrade the CI to that version due to the OpenSSL changes causing `webpack@4` builds to not work. I'd rather wait for that PR to be backported.